### PR TITLE
Remove toggl task

### PR DESCRIPTION
**Pull request by mtczerwinski**
I am waiting for https://github.com/kanet77/togglv8/issues/8.

### CODE REVIEW

File: app/models/toggl_integration/task.rb [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
 
-    def initialize(name:, pid:)
+    def initialize(id: nil, name:, pid:, wid: nil)
       @name = name
       @pid = pid
       @wid = wid
+      @id = id
+    end
+
+    def id?
```
Comment (lubieniebieski) [inline app/models/toggl_integration/task.rb]: where do we use it?

Comment (mtczerwinski) [inline app/models/toggl_integration/task.rb]: I would like to be concise with a design of Member or Team classes but frankly speaking, we don't need them and I should be prohibited to keep unused code and maybe some pushup as a punishment?

Comment (lubieniebieski) [inline app/models/toggl_integration/task.rb]: 10 will do

**Comment by lubieniebieski** [comment]
are we ok to update the gem @mtczerwinski?

**Comment by mtczerwinski** [comment]
Version `1.0.2` in not available at https://rubygems.org/gems/togglv8 but we can use a GitHub repository in this case.

**Comment by mtczerwinski** [comment]
Gem was updated to the newest version also, I fixed some bugs, but I have got a problem with my Toggl's subscription .

**Comment by lubieniebieski** [comment]
there's some error with codeclimate engine - it gives us some errors, but the code is good. merging anyway.

File: app/models/toggl_integration/api.rb [hunk 2]
```diff
@@ -105,6 +109,18 @@ def list_projects_users(team_id)
 
     private
 
+    def safe_fetch
```
Comment (mtczerwinski) [inline app/models/toggl_integration/api.rb]: Sly :)

**Pull request merged**